components/events/FaqSection.vue

```diff
@@ -0,0 +1,315 @@
+            <Applink v-bind="twitterQiskit">
+              follow Qiskit
+            </Applink>
+            and
+            <Applink v-bind="twitterIBMResearch">
+              IBM Research
+            </Applink>
+            on Twitter for the latest updates on new events, activities, and features on what is going on with the community. There is also a
+            <Applink v-bind="githubQiskit">
+              community Github
+            </Applink>
+            that you can explore and contribute to, and you can read regular
+            <Applink v-bind="mediumQiskit">
+              blog posts from our team and community members on Medium
+            </Applink>
+            . You can also
+            <Applink v-bind="qiskitSubscribe">
+              subscribe to the Qiskit YouTube
+            </Applink>
+            for the latest video content and our weekly live Seminar Series.
+          </p>
+
+          <p>
+            If you want to collaborate and connect with other members of the community, and participate in some of the ongoing conversations about quantum computing, you will want to
+            <Applink v-bind="joinqiskitslack">
+              check out our ever-growing Slack Community
+            </Applink>
+            . From there you can connect with current and past interns, IBM researchers, Qiskit advocates, and fellow community members directly.
+          </p>
+        </template>
+      </cv-accordion-item>
+
+      <cv-accordion-item>
+        <template #title>
+          Are there any upcoming events in my area that I can participate in?
+        </template>
+        <template #content>
+          <p>
+            All of our announced
+            <Applink v-bind="qiskitEvents">
+              upcoming events are listed on qiskit.org
+            </Applink>
+            , which is continually updated as we roll out more events throughout the year. You can also
+            <Applink v-bind="twitterQiskit">
+              follow Qiskit on Twitter
+            </Applink>
+            for the latest announcements on new and upcoming events!
+          </p>
+        </template>
+      </cv-accordion-item>
+
+      <cv-accordion-item>
+        <template #title>
+          Still have more questions?
+        </template>
+        <template #content>
+          <p>
+            For any questions about the summer school, please submit your questions using the form below. For all other enquiries, feel free to email us directly at
+            <Applink v-bind="mailtoQiskit">
+              qiskit.events@us.ibm.com
+            </Applink>.
+          </p>
+        </template>
+      </cv-accordion-item>
+    </cv-accordion>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+
+@Component
+export default class FaqSection extends Vue {
+  khanacademy = {
```

Comment:
We should type this with `GeneralLink` from `constants/appLinks.ts`.
```suggestion
  khanacademy: GeneralLink = {
```